packages/studio/package.json

```diff
@@ -10,54 +10,23 @@
     "dev": "tsc --watch --preserveWatchOutput -p tsconfig.node.json & tsc --watch --preserveWatchOutput -p tsconfig.json",
     "build": "run-script-os",
     "build:default": "npm run build:windows && chmod 755 lib/bin/studio.js",
-    "build:windows": "rimraf lib && tsc -p tsconfig.node.json && tsc -p tsconfig.json",
-    "preview": "vite preview",
-    "test": "jest",
-    "typecheck-jest": "npx tsc -p tests/tsconfig.json"
+    "build:windows": "rimraf lib && tsc -p tsconfig.node.json && tsc -p tsconfig.json"
   },
   "dependencies": {
-    "@dhmk/zustand-lens": "^2.0.5",
-    "@minoru/react-dnd-treeview": "^3.4.1",
-    "@restart/ui": "^1.5.2",
     "@vitejs/plugin-react": "^4.0.4",
     "@yext/studio-plugin": "0.22.0",
+    "@yext/studio-ui": "0.22.0",
     "autoprefixer": "^10.4.14",
     "cac": "^6.7.14",
-    "classnames": "^2.3.2",
     "cross-env": "^7.0.3",
-    "immer": "^9.0.21",
-    "lodash": "^4.17.21",
-    "path-browserify": "^1.0.1",
     "postcss": "^8.4.27",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
-    "react-modal": "3.16.1",
-    "react-toastify": "^9.1.1",
-    "react-tooltip": "^5.18.0",
-    "tailwind-merge": "^1.8.1",
-    "tailwindcss": "^3.3.3",
-    "vite": "^4.4.7",
-    "vite-plugin-svgr": "^2.4.0",
```

Comment:
do we still need `vite` and `vite-plugin-svgr` deps in `studio` for the Vite config?

Reply:
it will need to be added back in the following pr